### DISCUSSION TITLE
Provide Software Bill of Materials (SBOM) for Grafana

### DISCUSSION THREAD

**Discussion by Mario-Hofstaetter** [grafana/grafana#47550] [General]
TL;DR: Please provide a [software bill of materials (SBOM)](https://en.wikipedia.org/wiki/Software_bill_of_materials) for grafana to enable vulnerabilty & dependency tracking to users.

----

### What

We are using https://dependencytrack.org/ to track our own software components / dependencies and known vulnerabilites of them.
This is done by generating a sbom using https://cyclonedx.org/ in our builds and auto-uploading them to our dependency-track server.

Please provide a SBOM for Grafana releases containing it's dependencies. There are official generators for go and typescript, please see https://cyclonedx.org/tool-center/ 

### Why

Since grafana is accessible to users of our observability stack, it is inevitably at the risk of attacks (same goes for prometheus etc.).
Grafana team is doing an **awesome job** on handling CVEs and we try to update to latest stable versions regularly, but its not always possible to update immediately. So knowning when a particular version is at risk is better than not knowing.

It would be great if grafana provides a SBOM for all their products (we currently use grafana, loki, promtail).

Thank you.

**Comment by mellieA** [comment]
@Mario-Hofstaetter are you looking for an .spdx file for a release? 


For Grafana you can navigate to https://github.com/grafana/grafana/releases you'll see the files listed under Assets.

There is an issue [here](https://github.com/grafana/loki/pull/7397) to create the same workflow for Loki, but it's still in progress. 

![Screenshot 2023-01-18 at 1 04 09 PM](https://user-images.githubusercontent.com/18175287/213283068-e6d8c590-d78a-40c0-9c1d-b13fbd9e81d7.png)

**Reply by Mario-Hofstaetter** [reply]
@mellieA Exactly what I was looking for! I guess I missed this because it has been recently added, e.g. the latest 8.x.x release does not have the spdx file: https://github.com/grafana/grafana/releases/tag/v8.5.15 

But I can live with that and consider this discussion closed / resolved 👍
